### PR TITLE
Snvphyl

### PR DESCRIPTION
**Pull request by Takadonet**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by Takadonet** [comment]
All ready to be reviewed!

File: recipes/bcftools-snvphyl-plugin/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{% set version="1.5" %}
+about:
+  home: https://github.com/phac-nml/snvphyl-tools
+  license: apache_2.0
+  summary: The SNVPhyl (Single Nucleotide Variant PHYLogenomics) pipeline is a pipeline for identifying Single Nucleotide Variants (SNV) within a collection\
+   of microbial genomes and constructing a phylogenetic tree. This package is the bcftools C plugin
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/bcftools-snvphyl-plugin/meta.yaml]: 0

**Comment by bgruening** [comment]
Thanks @Takadonet!

File: recipes/bcftools-snvphyl-plugin/meta.yaml (hunk 1 above)
Comment (Takadonet) [reply recipes/bcftools-snvphyl-plugin/meta.yaml]: Fixed. Second time now that I forgot to change it to 0.

**Pull request merged**